### ISSUE TITLE
Grammar errors

### ISSUE THREAD

**Issue by vanslykej** [burrowss/about-me#1]
In the commit 6d8ec08862ead287b7702e506a0b5d08ad1d1324, there are a few grammar errors. In the sentence: "I began to help in on-line forums for help building computers" I would rephrase to something like "I began to participate in on-line forums for help in building computers." In the sentence following that one, "build" should be "built". In the last sentence of this paragraph, "its" should have an apostrophe. In the last paragraph I would change "From the first few days after arriving," to "Within the first few days of arriving".

**Comment by cookc3174** [comment]
Other than that, I thought it was pretty well written.

**Comment by vanslykej** [comment]
Agreed, the changes you made have made your writing more compelling.
